qiskit_sphinx_theme/pytorch_base/sidebar.html

```diff
@@ -27,6 +27,49 @@
   </div>
 </nav>
 
+<!-- PREVIOUS RELEASES URL -->
+{% if version_list %}
+<script>
+    const _ECOSYSTEM_PROJECTS = [
+        "aer",
+        "experiments",
+        "finance",
+        "machine-learning",
+        "nature",
+        "optimization",
+        "rustworkx",
+    ];
```

Comment:
Please double check if I am missing any projects. I took it from https://github.com/Qiskit/qiskit.org/issues/3038.

I'm not trying to handle providers or partners. I only am trying to fix projects in the format `qiskit.org/documentation/<project>`.

Reply:
missing `qiskit-dynamics` and `qiskit-metal`